src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate

```diff
@@ -1,7 +1,7 @@
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005">
   <TemplateData>
 	<Name>Cross-Platform App (Uno Platform)</Name>
-	<Description>Solution template for creating a cross-platform XAML app with the Uno Platform that targets UWP, Android, iOS, and WebAssembly</Description>
+	<Description>Solution Template for creating a multi-platform C# & XAML application targeting UWP, WebAssembly, Android, iOS, macOS and Linux</Description>
```

Comment:
```suggestion
	<Description>Template for creating a multi-platform C# & XAML application targeting UWP, WebAssembly, Android, iOS, macOS and Linux</Description>
```

Reply:
Removing the "Solution" @davidjohnoliver ?

Reply:
@sasakrsmanovic can verify if this is correct

Reply:
Linux is not available in the vsix templates for now.

Reply:
It is now :)